spec/request/service_credential_bindings_spec.rb

```diff
@@ -1392,7 +1392,18 @@ def check_filtered_bindings(*bindings)
               {
                 syslog_drain_url: syslog_drain_url,
                 credentials: credentials,
-                parameters: parameters
+                parameters: parameters,
+                service_id:'extra-field-service_id',
+                plan_id:'extra-field-plan_id',
+                name: 'extra-field-name',
+                bind_resource:{
+                  app_guid:'extra-field',
+                  space_guid:'extra-field'
+                },
+                context:
+                  {
+                    platform:'extra-field',
+                  }
```

Comment:
I think for the purpose of this test all we need is just an extra field `non_osbapi_param: 'foo'` even though having all the other ones doesn't harm

Reply:
I've changed it to just two fields: one that doesn't exist in our side so we verify it doesn't raise, and another one that does exist but we are not supposed to update to verify we also ignore those.